src/main/java/com/ibm/g11n/pipeline/client/NewResourceEntryData.java

```diff
@@ -0,0 +1,144 @@
+ * resource entry.
+ * 
+ * @author Yoshito Umaoka
+ */
+public class NewResourceEntryData {
+    private final String value;
+    private Boolean reviewed;
+    private Map<String, String> metadata;
+    private String partnerStatus;
+    private Integer sequenceNumber;
+
+    /**
+     * Constructor.
+     * 
+     * @param value The resource string value. Must not be null.
+     */
+    public NewResourceEntryData(String value) {
+        this.value = Objects.requireNonNull(value, "value must not be null");;
+    }
+
+    /**
+     * Returns the new string resource value.
+     * 
+     * @return The new string resource value.
+     */
+    public String getValue() {
+        return value;
+    }
+
+    /**
+     * Returns the new Boolean review status.
+     * 
+     * @return The Boolean review status.
+     */
+    public Boolean getReviewed() {
+        return reviewed;
+    }
+
+    /**
+     * Sets the new Boolean review status.
+     * 
+     * @param reviewed  The Boolean review status.
+     * @return This object.
+     */
+    public NewResourceEntryData setReviewed(Boolean reviewed) {
+        this.reviewed = reviewed;
+        return this;
+    }
+
+    /**
+     * Returns a map containing the new or updated key-value pairs.
+     * 
+     * @return A map containing the new or updated key-value pairs.
+     */
+    public Map<String, String> getMetadata() {
+        return metadata;
+    }
+
+    /**
+     * Sets a map containing the new or updated key-value pairs.
+     * <p>
+     * When a key currently exists in resource entry's metadata, the value
+     * of the key will be replaced with the new value. When a key
+     * currently exists and the new value is empty, the key-value
+     * pair will be removed. When a key does not exists, the key-value
+     * pair will be added.
+     * 
+     * @param metadata A map containing new or updated key-value pairs.
+     * @return This object.
+     */
+    public NewResourceEntryData setMetadata(Map<String, String> metadata) {
+        this.metadata = metadata;
+        return this;
+    }
+
+    /**
+     * Returns the new partner status string.
+     * 
+     * @return The new partner status string.
+     */
+    public String getPartnerStatus() {
+        return partnerStatus;
+    }
+
+    /**
+     * Sets the new partner status string.
+     * <p>
+     * This property is reserved for a translation partner's use.
+     * If you don't use any translation partners, you can maintain
+     * custom translation status string with this property.
+     * 
+     * @param partnerStatus The new partner status string.
+     * @return This object.
+     */
+    public NewResourceEntryData setPartnerStatus(String partnerStatus) {
+        this.partnerStatus = partnerStatus;
+        return this;
+    }
+
+    /**
+     * Returns the sequence number.
+     * 
+     * @return The sequence number.
+     */
+    public Integer getSequenceNumber() {
+        return sequenceNumber;
```

Comment:
getSequenceNumber? If so, null is returned in this case. The sequence number is optional.